src/content/docs/en/pages/cli/azioncli-beta/azioncli-beta.mdx

```diff
@@ -0,0 +1,1848 @@
+
+The CLI currently provides three templates and the type `cdn` for building edge applications that follow `--type`:
+
+- Static
+
+```bash
+    azioncli edge_applications init --name <"application_name"> --type static
+```
+
+- Next.js
+
+```bash
+    azioncli edge_applications init --name <"application_name"> --type nextjs
+```
+
+- CDN
+
+```bash
+    azioncli edge_applications init --name <"application_name"> --type cdn
+```
+
+> **Note**: if there's no `package.json` configuration file in your edge application directory, the CLI will return the following error message:
+
+```bash
+    Error: Failed to find package.json in current directory. Verify if you are currently in your project's directory
+```
+
+In that case, create a `package.json` configuration file for your project in the same directory as your edge application. Then rerun the command in order to initialize the application.
+
+> **Tip**: the best solution for creating a `package.json` file is to start a Node.js project in your edge application directory. To do so, use the command below from the Node.js runtime environment:
+>
+> `npm init -y`
+
+Command response in terminal:
+
+```bash
+    Wrote to /Users/<user_name>/web-app-proj/package.json:
+
+    {
+    "name": "web-app-proj",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC"
+    }
+```
+
+##### No
+
+The `--no` option forces a `no` response to all user inputs.
+
+#### Yes
+
+The `--yes` option forces a 'yes' response to all user inputs.
+
+---
+
+#### Building
+
+The second step in creating an edge application is to build it based on the attributes provided in the `init` command. The command used to build the application is the `build` command.
+
+##### Usage
+
+```bash
+    azioncli edge_applications build
+```
+
+##### Flags
+
+##### Help
+
+The only flag available for the `build` command is `--help`, which displays a brief help screen.
+
+**Note**: a requirement for using the `build` command is the presence of the [webPack](https://webpack.js.org/) framework installed on your computer.
+
+---
+
+#### Publishing
+
+The final step in creating an edge application is publishing it on the Azion platform. The `publish` command performs this task automatically.
+
+##### Usage
+
+```bash
+    azioncli edge_applications publish
+```
+
+##### Flags
+
+##### Help
+
+The only flag available for the `publish` command is `--help`, which displays a brief help screen.
+
+**Note**: publishing an edge application via **RTM** or CLI isn't an instantaneous process. There's a time to propagate and configure the application in Azion's edge locations. It may be necessary to wait a few minutes for the URL to be activated and the application page to be effectively displayed in the browser.
```

Comment:
```suggestion
:::note
Publishing an edge application via **RTM** or CLI isn't an instantaneous process. There's a time to propagate and configure the application in Azion's edge locations. It may be necessary to wait a few minutes for the URL to be activated and the application page to be effectively displayed in the browser.
:::
```